components/stacks-network/src/orchestrator.rs

```diff
@@ -1421,6 +1440,75 @@ events_keys = ["*"]
         ctx.try_log(|logger| slog::info!(logger, "Created container subnet-node: {}", container));
         self.subnet_node_container_id = Some(container.clone());
 
+        // Download the subnet contract(s) from the container and return them on success
```

Comment:
Can you update this comment to say something like `Download the subnet contract(s) from the subnet container, and write them to $path`, where you replace `$path` with a description of where it gets written.

Reply:
Yup, sounds good.

"Download the subnet contract(s) from the container and write them to Clarinet's cache directory."